Sharpasonne/Players.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using JetBrains.Annotations;
+using Optional;
+using Sharpasonne.GameActions;
+using Sharpasonne.Models;
+using Sharpasonne.Rules;
+
+namespace Sharpasonne
+{
+    public class Players
+    {
+        /// <summary>
+        /// Least number of players allowed to create a <see cref="Players" /> instance.
+        /// </summary>
+        public const int LOWER_BOUND = 2;
```

Comment:
Is this the usual casing for consts?

Reply:
Not sure, as you said earlier we need StyleCop or similar 👍